packages/@uppy/dashboard/src/components/Slide.js

```diff
@@ -23,7 +23,6 @@ class Slide extends Component {
     }
   }
 
-  // TODO: refactor this component to not use componentWillUpdate
```

Comment:
Hum why remove this TODO? `componentWillUpdate` is still deprecated – it's been renamed to `UNSAFE_componentWillUpdate` in React 17: https://reactjs.org/docs/react-component.html#unsafe_componentwillupdate